admin_guide/managing_networking.adoc

```diff
@@ -903,3 +903,50 @@ in all other projects. The project to be isolated must first be configured to
 opt in to isolation by setting the proper annotation on its Namespace object,
```

Comment:
The paragraph here needs to be updated too... part of it isn't visible in this patch. But in the sentence "network isolation is controlled entirely by `NetworkPolicy` objects and the associated Namespace annotation", you want to remove the words "and the associated Namespace annotation". But keep the external link, just have it on the words "`NetworkPolicy` objects". Then remove the reference to "the proper annotation" in the following sentence as well (which probably means totally rewriting it).